handlers/dimension/handler.go

```diff
@@ -1,77 +1,19 @@
 package dimension
 
 import (
-	"encoding/json"
 	"net/http"
-
-	"github.com/ONSdigital/dp-dd-api-stub/models"
 )
 
 func Handler(w http.ResponseWriter, req *http.Request) {
-
-	optionsEnglandNorthEast := make([]*models.DimensionOption, 4)
-	optionsEnglandNorthEast[0] = &models.DimensionOption{
-		ID:   "2000017",
-		Name: "Hartlepool",
-	}
-	optionsEnglandNorthEast[1] = &models.DimensionOption{
-		ID:   "2000018",
-		Name: "Middlesbrough",
-	}
-	optionsEnglandNorthEast[2] = &models.DimensionOption{
-		ID:   "2000019",
-		Name: "Redcar and Cleveland",
-	}
-	optionsEnglandNorthEast[3] = &models.DimensionOption{
-		ID:   "2000020",
-		Name: "Stockton-on-Tees",
-	}
-
-	optionsEngland := make([]*models.DimensionOption, 4)
-	optionsEngland[0] = &models.DimensionOption{
-		ID:      "E12000001",
-		Name:    "North East",
-		Options: optionsEnglandNorthEast,
-	}
-	optionsEngland[1] = &models.DimensionOption{
-		ID:   "E12000002",
-		Name: "North West",
-	}
-	optionsEngland[2] = &models.DimensionOption{
-		ID:   "E12000003",
-		Name: "Yorkshire and The Humber",
-	}
-	optionsEngland[3] = &models.DimensionOption{
-		ID:   "E12000004",
-		Name: "East Midlands",
+	view := req.URL.Query().Get("view")
+	switch view {
+	case "hierarchy":
+		hierarchyHandler(w, req)
+		break
+	case "list":
```

Comment:
feels like this should be more explicit - i.e., handle `"hierarchy"`, `"list"` and `""` (default to list), with everything else being a `400 Bad Request` error

makes it clearer to an API user if they've passed in an invalid value